src/join_variant_summary_with_clinvar_alleles.py

```diff
@@ -100,6 +119,9 @@ def join_variant_summary_with_clinvar_alleles(
     df['conflicted'] = df['original_clnsig'].str.contains(
         "onflicting", case=False).astype(int)
 
+    df.loc[df['conflicted'] == 1, 'clnsig'] = df[df['conflicted'] == 1].apply(determine_most_pathogenic_submission,
+                                                                              axis=1)
```

Comment:
Is the `axis` parameter correct here? If you're applying the `determine_most_pathogenic_submission` to each row, shouldn't it be `axis=0` (the default)?

A minor improvement to performance/readability may be to factor out the Boolean index `df['conflicted'] == 1`. E.g.:
```python
conflicted_rows = df['conflicted'] == 1
df.loc[conflicted_rows, 'clnsig'] = df[conflicted_rows, 'clnsig'].apply(determine_most_pathogenic_submission,                                                                               axis=1)
```